ui/src/common/utils.js

```diff
@@ -3,8 +3,8 @@ import cloneDeep from 'lodash.clonedeep';
 import moment from 'moment-timezone';
 import { Map } from 'immutable';
 import NumberAbbreviator from 'number-abbreviate';
+import { saveAs } from 'file-saver';
```

Comment:
i don't think this brings anything valuable at the moment, so we can keep our simple function.

Reply:
It seemed cleaner and easier since it would work in every browser and display the correct filename and extension.
Otherwise, I'm not sure why we have different code for Safari than for the rest. Why weren't we setting the filename and extension for the rest of the browsers? If we just use that same code for Chrome it works perfectly.

Reply:
"the rest" code was there before everything else, and setting file name was never strong requirement. If filename setting is important you can just keep `IE` and `anchor` based implementation.

Our code works for the browsers we officially support already, and this package does more to handle other cases that we don't care. And there is no guarantee it will load some heavy dependencies on a later release. Therefore it's not worth it.